01-html/03-caitlinandcollin.html

```diff
@@ -2,4 +2,228 @@
+      <img class="inline-block" src="http://img.mota.ru/upload/wallpapers/source/2011/04/11/10/02/25043/mota_ru_1041116.jpg" alt="Beach people" width="300">
+      <p>Well, Caitlin thought it was romantic nonetheless.</p>
+      <p>With the sun rising over the ocean, whales breaching in unison in the distance, and turtles swimming in the surf below, Caitlin said yes! And after she was able to breathe again, and the tears and shaking stopped, they kissed - and lived happily
+        ever after.</p>
+    </article>
+  </section>
+  <section>
+    <header id="ceremony">CEREMONY & RECEPTION</header>
+    <article class="inline-block">
+      <h2>4:00<sup><small>PM</small></sup>, NOVEMBER 15, 2014</h2>
+      <p>This afternoon ceremony and reception will take place at the same location in Seattle's historic Pioneer Square neighborhood, right in downtown Seattle. We chose this location for a few reasons. First of all, because it is breathtaking, soulful,
+        and perfectly sized. It is also the previous site of the Elliot Bay Bookstore, which we were both raised to love.</p>
+    </article>
+    <article class="inline-block">
+      <img src="https://s-media-cache-ak0.pinimg.com/236x/4d/bf/21/4dbf216f8878559127a0afbb34baa642.jpg" alt="venue">
+      <p>Now refurbished by Urban Light Studios, Axis is a killer modernized venue, but with the age and emotion of something more substantial. Look for fun touches, like the markings on the floor or the paper towel holders in the restrooms. A note: we feel
+        so blessed to have so many loved ones that we will be tight on space, but due to that that, any dates or guests not named on the invitation must be cleared with us at time of RSVP. Also, while we love children, we ask that you leave yours at home
+        for the night. Thank you!</p>
+    </article>
+    <article class="inline-block">
+      <iframe src="https://www.google.com/maps/place/308+1st+Ave+S,+Seattle,+WA+98104/@47.5998111,-122.3363174,17z/data=!3m1!4b1!4m2!3m1!1s0x54906aa520aedad3:0xb4bba6560c0f95fe?hl=en-US" alt="Map of Pioneer Square"></iframe>
+      <p class="font-enlarge">JOIN US FOR THE RECEPTION</p>
+    </article>
+    <article class="inline-block">
+      <address>
+        <span class="enlarge-font">AXIS</span>
+        <br> 308 1st Avenue S
+        <br> Pioneer Square
+        <br> Seattle, WA 98104</address>
+      <p>We hope that having all parts of our wedding in one central location will give you time to be present for our ceremony, relax for cocktail hour, and indulge for the reception - and then easily be able to find your way home.</p>
+    </article>
+  </section>
+  <header id="wedding-party">THE
+    <span class="enlarge-font">WEDDING PARTY</span>
+  </header>
+  <section class="inline-block">
+    <article>
+      <p>
+        <img src="http://www.cinemablend.com/images/news_img/92797/GWENDOLINE_CHRISTIE_92797.jpg" alt="tarth" width="300">
+        <span class="name">BREEZY GRIFFITH</span>
+      </p>
```

Comment:
Unless you've got a paragraph of text here, you might want to wrap this in a `div` instead of `p`.